### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by naveen246**

### CODE REVIEW

File: mem/memtable.go [hunk 1]
```diff
@@ -11,11 +12,6 @@ import (
 	"sync/atomic"
 )
 
-type DeleteRange struct {
```
Comment (naveen246) [inline mem/memtable.go:14]: This is not used anywhere

File: mem/memtable.go [hunk 2]
```diff
@@ -93,12 +89,10 @@ func (m *Memtable) Write(batch writeBatch) (bool, error) {
 	var err error
 	batch.Range(func(key []byte, value []byte) bool {
 		if err = writeIter.Add(key, value, 0); err != nil {
-			if //goland:noinspection GoDirectComparisonOfErrors
-			err == arenaskl2.ErrRecordExists {
+			if errors.Is(arenaskl2.ErrRecordExists, err) {
```
Comment (purplefox) [inline mem/memtable.go:92]: We deliberately use exact comparison here as this is performance sensitive code

Comment (naveen246) [reply mem/memtable.go]: ok, got it, changed it back to original code

Comment (stanislavkozlovski) [reply mem/memtable.go]: maybe this can be marked in the comment alongside GoDirectComparisonOfErrors ?

**Pull request merged**